README.md

```diff
@@ -7,6 +7,7 @@ Requirements:
 - XeLaTeX
 - latexmk
 - Fira fonts
+- pygmentize
```

Comment:
Maybe replace that with "pygments"? That's the name of the package on most systems :)